pkg/apis/serving/k8s_validation.go

```diff
@@ -82,8 +82,9 @@ var (
 )
 
 // ValidateVolumes validates the Volumes of a PodSpec.
-func ValidateVolumes(ctx context.Context, vs []corev1.Volume, mountedVolumes sets.String) (sets.String, *apis.FieldError) {
+func ValidateVolumes(ctx context.Context, vs []corev1.Volume, mountedVolumes sets.String) (sets.String, sets.String, *apis.FieldError) {
```

Comment:
since we have two sets.String in the return section now I'd probably add variable names for them and maybe add to the method comment to indicate what each is

Reply:
Yes ok.